lib/jekyll/converters/markdown/redcarpet_parser.rb

```diff
@@ -47,7 +47,7 @@ def block_code(code, lang)
   end
 
   module WithRouge
-    def block_code(code, lang)
+    def block_code(_code, lang)
```

Comment:
```text
lib/jekyll/converters/markdown/redcarpet_parser.rb:50:20: W: Lint/UnusedMethodArgument: Unused method argument - code. If it's necessary, use _ or _code as an argument name to indicate that it won't be used.
    def block_code(code, lang)
                   ^^^^
```

Reply:
Hmm... this looks like a Rubocop bug, as `code` is clearly used in line 53.

Reply:
@pathawks it doesn't look like a Rubocop bug to me since if you look at `line 51`, `code` is being **immediately re-assigned** with `code = "<pre>#{super}</pre>"` so in effect overriding whatever has been passed to `:block_code` as the first argument..

Reply:
You are right! It is doing _exactly_ what it’s supposed to do. We don’t need this.